core/variables.ts

```diff
@@ -82,11 +82,10 @@ export function allDeveloperVariables(workspace: Workspace): string[] {
   const variables = new Set<string>();
   for (let i = 0, block; block = blocks[i]; i++) {
     let getDeveloperVariables = block.getDeveloperVariables;
-    if (!getDeveloperVariables &&
-        (block as AnyDuringMigration).getDeveloperVars) {
+    if (!getDeveloperVariables && (block as any).getDeveloperVars) {
       // August 2018: getDeveloperVars() was deprecated and renamed
       // getDeveloperVariables().
-      getDeveloperVariables = (block as AnyDuringMigration).getDeveloperVars;
+      getDeveloperVariables = (block as any).getDeveloperVars;
       if (!ALL_DEVELOPER_VARS_WARNINGS_BY_BLOCK_TYPE[block.type]) {
         console.warn(
```

Comment:
Can you replace this with a proper deprecate call w/ a date?

Reply:
Will just remove it in a separate PR.